modules/docs/markdown/04-codegen/01-customisation/08-wildcard.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_label: Wildcard imports
+title: Adjust Scala wildcard
+---
+
+Scala has a specific syntax for wildcard argument in types. In Scala 2, that was the underscore: `_`. But with Scala 3, this is changing. See this [page](https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html) for more information.
```

Comment:
**nitpick:** I've heard links with labels like `this`, `page`, etc. are bad for accessibility - IIRC the issue is that screen readers won't give any useful label, and users will be forced to listen to the entire URL and figure out themselves what the target is.

Also, these labels tend to be kinda useless for everyone else as well...

I would suggest being more descriptive about the target:

```suggestion
Scala has a specific syntax for wildcard argument in types. In Scala 2, that was the underscore: `_`. But with Scala 3, this is changing. See [the language reference page](https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html) for more information.
```